### PR TITLE
Version 2

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by alexAubin** [comment]
!testme

File: scripts/install [hunk 1]
```diff
@@ -10,64 +10,6 @@ source _common.sh
...
-# STANDARD MODIFICATIONS
-#=================================================
-# INSTALL DEPENDENCIES
-#=================================================
-ynh_script_progression --message="Installing dependencies..."
-
-ynh_install_app_dependencies $pkg_dependencies
-ynh_install_apps --apps="$app_dependencies"
-
 #=================================================
```
Comment (alexAubin) [inline scripts/install:13]:
```suggestion
ynh_install_apps --apps="gemserv"
#=================================================
```

File: manifest.toml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+    [resources.system_user]
+
+    [resources.install_dir]
+
+    [resources.permissions]
+    main.url = "/"
+
+    [resources.apt]
+    packages = "gemserv, php8.2-fpm, php8.2-mbstring"
```
Comment (alexAubin) [inline manifest.toml]:
```suggestion
    packages = "php8.2-fpm, php8.2-mbstring"
```

**Comment by alexAubin** [comment]
!testme

**Comment by Salamandar** [comment]
!testme

**Pull request merged**